wondrous-app/components/Settings/Payouts/styles.tsx

```diff
@@ -1,11 +1,268 @@
+))`
+  &.MuiPaper-root {
+    background: ${palette.grey900};
+    width: 100%;
+    max-width: 270px;
+    color: ${palette.white};
+    border: 1px solid ${palette.grey75};
+    transform: translateX(28%) !important;
+  }
+
+  &.MuiPaper-root > .MuiList-root {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    max-height: 200px;
+  }
+
+  &.MuiPaper-root > .MuiList-padding {
+    padding: 8px;
+  }
+
+  ::-webkit-scrollbar {
+    width: 6px;
+  }
+
+  ::-webkit-scrollbar-track {
+    background: ${palette.grey10};
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    background: ${palette.grey50};
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+  }
+
+  ::-webkit-scrollbar-thumb:hover {
+    background: ${palette.grey76};
+  }
+`;
+
+export const PayoutSelectionSelectMenuItem = styled(MenuItem)`
+  && {
+    background: ${palette.black98};
+    color: ${(props) => (props.isSelected ? palette.white : palette.grey250)};
+    border: 1px solid;
+    border-color: ${(props) => (props.isSelected ? palette.highlightPurple : palette.black98)};
+    border-radius: 4px;
+    font-weight: ${(props) => (props.isSelected ? 500 : 400)};
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    padding: 10px;
+    transition: background 0.2s ease-out;
+
+    :hover {
+      background: ${palette.black92};
+    }
+  }
+`;
+
+export const PayoutSelectionSelectValueDisplay = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`;
+
+export const PayoutSelectionSelectValueDisplayText = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 19.14px;
+    color: ${(props) => (props.isActive ? palette.white : palette.grey57)};
+  }
+`;
+
+export const PayoutCount = styled(Typography)`
+  && {
+    font-family: ${typography.fontFamily};
+    font-size: 14px;
+    font-weight: 500;
+    color: ${palette.blue20};
+  }
+`;
+
+export const StyledTableContainer = styled(TableContainer)`
+  && {
+    width: 100%;
+    overflow-x: visible;
+    min-width: 890px;
+  }
+`;
+
+export const StyledTable = styled(Table)`
+  && {
+    width: 100%;
+    margin: 14px 0;
+    border-collapse: collapse;
+  }
+`;
+
+export const StyledTableHead = styled(TableHead)`
+  & .MuiTableCell-head {
+    color: #ccbbff;
+    background: ${palette.black97};
+    border: none;
+    font-family: 'Space Grotesk';
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    letter-spacing: 0em;
+    border-radius: 0px;
+    padding: 10px !important;
+  }
+
+  & .MuiTableCell-head:first-of-type {
+    border-top-left-radius: 6px;
+    border-bottom-left-radius: 6px;
+  }
+
+  & .MuiTableCell-head:last-of-type {
+    border-top-right-radius: 6px;
+    border-bottom-right-radius: 6px;
+  }
+`;
+
+export const StyledTableBody = styled(TableBody)`
```

Comment:
you don't need styled component, just write `<TableBody border="none" mt="15px">`

Reply:
doesn't support border / margin props